docs/changes.xml

```diff
@@ -29,6 +29,11 @@ NGINX Unit updated to 1.33.0.
 
 </changes>
 
+<change>
+<para>
+wasm and wasm-wasi-component now built against libwasmtime 24.0.0
+</para>
+</change>
 
```

Comment:
I don't think we really need this entry, it shouldn't really be a user visible thing... I would just drop this change.